src/pyOpenMS/pxds/DigestionEnzyme.pxd

```diff
@@ -8,20 +8,20 @@ cdef extern from "<OpenMS/CHEMISTRY/DigestionEnzyme.h>" namespace "OpenMS":
     
     cdef cppclass DigestionEnzyme "OpenMS::DigestionEnzyme":
         DigestionEnzyme(DigestionEnzyme) nogil except +
-        DigestionEnzyme(const String & name, const String & cleavage_regex, libcpp_set[ String ] & synonyms, String regex_description) nogil except +
-        void setName(const String & name) nogil except +
+        DigestionEnzyme(String name, String cleavage_regex, libcpp_set[ String ] & synonyms, String regex_description) nogil except +
+        void setName(String name) nogil except +
         String getName() nogil except +
         void setSynonyms(libcpp_set[ String ] & synonyms) nogil except +
         void addSynonym(const String & synonym) nogil except +
         libcpp_set[ String ] getSynonyms() nogil except +
-        void setRegEx(const String & cleavage_regex) nogil except +
+        void setRegEx(String cleavage_regex) nogil except +
         String getRegEx() nogil except +
-        void setRegExDescription(const String & value) nogil except +
+        void setRegExDescription(String value) nogil except +
         String getRegExDescription() nogil except +
         bool operator==(const DigestionEnzyme & enzyme) nogil except +
         bool operator!=(const DigestionEnzyme & enzyme) nogil except +
-        bool operator==(const String & cleavage_regex) nogil except +
-        bool operator!=(const String & cleavage_regex) nogil except +
+        bool operator==(String cleavage_regex) nogil except +
```

Comment:
I think this is not possible with autowrap.
You can do a wrapas "equalsRegex" or leave it out.
Same for !=

Reply:
I mean the overloaded comparison. This overwrites the old/correct/usual comparator